appservice/src/connectToGitHub.ts

```diff
@@ -61,15 +62,15 @@ export async function connectToGitHub(node: IAzureNode, client: SiteClient): Pro
         repoQuickPick = (await ext.ui.showQuickPick(repoQuickPicks, { placeHolder: 'Choose project.' })).data;
 
         if (repoQuickPick.url === requestOptions.nextLink) {
-            requestOptions.url = requestOptions.nextLink;
+            requestOptions.url = nonNullProp(requestOptions, 'nextLink');
             // remove the stale Load More quick pick
             repoQuickPicks.pop();
             const moreGitHubRepos: Object[] = await getJsonRequest(requestOptions, node);
             repoQuickPicks = repoQuickPicks.concat(createQuickPickFromJsons(moreGitHubRepos, 'name', undefined, ['url', 'html_url']));
         } else {
             repoSelected = true;
```

Comment:
Where's repoSelected  defined?

Reply:
Second line of this `connectToGitHub` function. I changed it from a 'while' to a 'do/while' because I was getting this error:
![screen shot 2018-10-01 at 11 49 54 am](https://user-images.githubusercontent.com/11282622/46309113-4cc83b80-c570-11e8-83ed-9ac00d5c7dcc.png)

Even though the while loop would've always been run once since `repoSelected` is set to false by default.